Week1/homework/js-exercises/logRemainder.js

```diff
@@ -0,0 +1,18 @@
+'use script';
+
+{
+  if ((x = 7)) {
```

Comment:
Ah. This logic works but it is not a good idea to do an assignment operation inside an `if` block. Could you please revisit this?

Also, if you look carefully, the assignment says:
```
1. If x equals 7, and the only other statement is x = x % 3, what would be the value of x after the calculation?
2. If y equals 21, and the only other statement is x = x % 4, what would be the value of y after the calculation?
3. If z equals 13, and the only other statement is x = x % 2, what would be the value of z after the calculation?
```

Have a careful look 😉